### DISCUSSION TITLE
Need help to install and operate please

### DISCUSSION THREAD

**Discussion by jasonromney** [akosbalasko/yarle#143] [Q&A] [answered]
Thanks for creating this tool - probably what many people are seeking. 
But I'm also in the position where I've tried and failed to install this. I'd like to become a user to migrate from EN to Roam though, and I think I'm very close, but just need a bit more help to finish the install.
Can you provide more specific detail please on where the configuration file needs to be placed, and/or how to reference its location in the config line in terminal. I've tried many permutations and combinations of syntax to get the right terminal command but need a quick tutorial on exactly how to parse a particular location (such as my /Documents folder). I know this should be really simple... Thanks, JR

**Comment by Cito** [comment, marked as answer]
> where the configuration file needs to be placed

You can specify that yourself on the command line with the `--configFile` switch. E.g. if your config file is `convert.json` in the current directory where you run the command, you would use `--configFile convert.json` or `--configFile ./convert.json`.

**Reply by akosbalasko** [reply]
@Cito , my friend, you're faster than me! :)
